### PR TITLE
AST-36690 | cleanup integration test

### PR DESCRIPTION
**Pull request by tamarleviCm**
### Description

> deleting the projects that created by the tests
> adding ApplicationIds to project command result print

### References

> https://checkmarx.atlassian.net/browse/AST-36690

### Testing

> check if the created project connected to an application

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR (if applicable).
- [ ] I have updated the CLI help for new/changed functionality in this PR (if applicable).
- [x] All active GitHub checks for tests, formatting, and security are passing
- [x] The correct base branch is being used

### CODE REVIEW

File: test/integration/scan_test.go [hunk 1]
```diff
@@ -82,7 +82,7 @@ func TestScanCreate_ExistingApplicationAndExistingProject_CreateScanSuccessfully
 	assert.NilError(t, err)
 }
 
-func TestScanCreate_ExistingApplicationAndNotExistingProject_CreatingNewProjectAndCreateScanSuccessfully(t *testing.T) {
+func TestScanCreateWithApplication(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (tamarleviCm) [reply test/integration/scan_test.go:85]: It's really nice but it's the only 3 tests out of hundreds that use the convention.

File: test/integration/project_test.go [hunk 2]
```diff
@@ -106,12 +106,14 @@ func TestProjectCreate_ApplicationDoesntExist_FailAndReturnErrorMessage(t *testi
...
+	err, outBuffer := executeCommand(
 		t, "project", "create", flag(params.FormatFlag),
 		printer.FormatJSON, flag(params.ProjectName), projectNameRandom,
 		flag(params.ApplicationName), "my-application",
 	)
-
+	createdProject := wrappers.ProjectResponseModel{}
+	unmarshall(t, outBuffer, &createdProject, "Reading project create response JSON should pass")
+	defer deleteProject(t, createdProject.ID)
 	assert.NilError(t, err)
```
Comment (OrShamirCM) [inline test/integration/project_test.go:117]: please add assert as you did in the other test

File: test/integration/project_test.go [hunk 3]
```diff
@@ -106,12 +106,14 @@ func TestProjectCreate_ApplicationDoesntExist_FailAndReturnErrorMessage(t *testi
...
 func TestProjectCreate_ApplicationExists_CreateProjectSuccessfully(t *testing.T) {
 
-	err, _ := executeCommand(
+	err, outBuffer := executeCommand(
 		t, "project", "create", flag(params.FormatFlag),
 		printer.FormatJSON, flag(params.ProjectName), projectNameRandom,
 		flag(params.ApplicationName), "my-application",
 	)
-
+	createdProject := wrappers.ProjectResponseModel{}
```
Comment (OrShamirCM) [inline test/integration/project_test.go:114]: there is in assertion on the project creation there?

File: test/integration/project_test.go (hunk 2 above)
Comment (tamarleviCm) [reply test/integration/project_test.go:117]: done

File: test/integration/project_test.go (hunk 3 above)
Comment (tamarleviCm) [reply test/integration/project_test.go:114]: there is an error and it checked in line117

Comment (OrShamirCM) [reply test/integration/project_test.go:114]: we should have an assertion and not just error... can you add please? (if it's not complex)

Comment (tamarleviCm) [reply test/integration/project_test.go:114]: done

**Review by OrShamirCM — APPROVED** [review]

**Pull request merged**